CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `Field`, `CryptoHash`, `StorageValue` and `ExonumJson` traits have been
   implemented for `chrono::Duration` structure. (#653)
 
+- Added a method `execute` in `Service` trait. (#667)
```

Comment:
`execute method has been added to the Service trait`

Reply:
As for me the past simple tense is more appropriate here.

Reply:
We're using present perfect everywhere else though